config/database.yml

```diff
@@ -58,6 +60,8 @@ development:
 test:
   <<: *default
   database: Recipe_app_test
+  username: postgres
+  password: Epal4postgres.
```

Comment:
[OPTIONAL]
Since We are working together, I think It would be great to leave such parameters which may affect both of us. Because As I know, here the `username` and `password` are supposed to be  `The password associated with the Postgres role (username)` in our local environment.

Reply:
Noted